stores/amazon_monitoring.py

```diff
@@ -297,6 +334,10 @@ async def stock_check(self, queue: asyncio.Queue, future: asyncio.Future):
 
             check_count += 1
 
+            if self.delay == 0: 
```

Comment:
Why would delay = 0?

Reply:
I first set it so that you could "activate" rotation mode but since proxies.json structure changed entirely I could just get rid of it and incorporate self.delay into the cooldown checker.